core/modules/parsers/wikiparser/rules/syslink.js

```diff
@@ -0,0 +1,45 @@
+/*\
+title: $:/core/modules/parsers/wikiparser/rules/syslink.js
+type: application/javascript
+module-type: wikirule
+
+Wiki text inline rule for system tiddler links.
+Can be suppressed preceding them with `~`.
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.name = "syslink";
+exports.types = {inline: true};
+
+exports.init = function(parser) {
+	this.parser = parser;
+	// Regexp to match
+	this.matchRegExp = /~?(?:\$):[^\s<>{}\[\]`|'"\\^~]+(?:\/|\b)/mg;
```

Comment:
Not a crash (no js error), only the titles are jumbled up...

![jumbled](https://cloud.githubusercontent.com/assets/97566/5692790/7c3e6dc2-9904-11e4-998b-63fd7a9f059a.jpg)

So, this doesn't work...

`this.matchRegExp = /~?\$:[^\s|]+(?:\/|\b)/mg;`

This works...

`this.matchRegExp = /~?\$:[^\s<>|]+(?:\/|\b)/mg;`